create_cluster.py

```diff
@@ -566,14 +572,15 @@ def print_failure_message():
 @click.option('--environment', '-e', multiple=True)
 @click.option('--sns-topic', help='SNS topic name to send Auto-Recovery notifications to')
 @click.option('--sns-email', help='Email address to subscribe to Auto-Recovery SNS topic')
+@click.option('--instance-profile-name', help='Instance profile name to be used, optional')
```

Comment:
Other options don't mention 'optional'.  Could we update README and mention it there somehow?